metrics/metrics.go

```diff
@@ -38,22 +37,14 @@ var (
 	APIRequestDuration = stats.Float64("api/request_duration_ms", "Duration of API requests", stats.UnitMilliseconds)
 
 	// rcmgr
-	RcmgrConn           = stats.Int64("rcmgr/conn", "Number of connections", stats.UnitDimensionless)
-	RcmgrStream         = stats.Int64("rcmgr/stream", "Number of allowed streams", stats.UnitDimensionless)
-	RcmgrAllowConn      = stats.Int64("rcmgr/allow_conn", "Number of allowed connections", stats.UnitDimensionless)
-	RcmgrBlockConn      = stats.Int64("rcmgr/block_conn", "Number of blocked connections", stats.UnitDimensionless)
-	RcmgrAllowStream    = stats.Int64("rcmgr/allow_stream", "Number of allowed streams", stats.UnitDimensionless)
-	RcmgrBlockStream    = stats.Int64("rcmgr/block_stream", "Number of blocked streams", stats.UnitDimensionless)
-	RcmgrAllowPeer      = stats.Int64("rcmgr/allow_peer", "Number of allowed peer connections", stats.UnitDimensionless)
-	RcmgrBlockPeer      = stats.Int64("rcmgr/block_peer", "Number of blocked peer connections", stats.UnitDimensionless)
-	RcmgrAllowProto     = stats.Int64("rcmgr/allow_proto", "Number of allowed streams attached to a protocol", stats.UnitDimensionless)
-	RcmgrBlockProto     = stats.Int64("rcmgr/block_proto", "Number of blocked blocked streams attached to a protocol", stats.UnitDimensionless)
-	RcmgrBlockProtoPeer = stats.Int64("rcmgr/block_proto", "Number of blocked blocked streams attached to a protocol for a specific peer", stats.UnitDimensionless)
-	RcmgrAllowSvc       = stats.Int64("rcmgr/allow_svc", "Number of allowed streams attached to a service", stats.UnitDimensionless)
-	RcmgrBlockSvc       = stats.Int64("rcmgr/block_svc", "Number of blocked blocked streams attached to a service", stats.UnitDimensionless)
-	RcmgrBlockSvcPeer   = stats.Int64("rcmgr/block_svc", "Number of blocked blocked streams attached to a service for a specific peer", stats.UnitDimensionless)
-	RcmgrAllowMem       = stats.Int64("rcmgr/allow_mem", "Number of allowed memory reservations", stats.UnitDimensionless)
-	RcmgrBlockMem       = stats.Int64("rcmgr/block_mem", "Number of blocked memory reservations", stats.UnitDimensionless)
+	RcmgrConn      = stats.Int64("rcmgr/conn", "Number of connections", stats.UnitDimensionless)
+	RcmgrStream    = stats.Int64("rcmgr/stream", "Number of allowed streams", stats.UnitDimensionless)
+	RcmgrPeer      = stats.Int64("rcmgr/peer", "Number of peers", stats.UnitDimensionless)
+	RcmgrProto     = stats.Int64("rcmgr/proto", "Number of allowed streams attached to a protocol", stats.UnitDimensionless)
+	RcmgrProtoPeer = stats.Int64("rcmgr/proto", "Number of allowed streams attached to a protocol for a specific peer", stats.UnitDimensionless)
+	RcmgrSvc       = stats.Int64("rcmgr/svc", "Number of streams attached to a service", stats.UnitDimensionless)
+	RcmgrSvcPeer   = stats.Int64("rcmgr/svc", "Number of streams attached to a service for a specific peer", stats.UnitDimensionless)
```

Comment:
```suggestion
```
Monitoring of peer_id stuff needs to be done by logging not prometheus metrics.

Reply:
resolved on the latest commit.